mantidproject/mantid
# Imaging GUI: fix submission of custom commands

Fixes the issue and adds a regression test as part of a battery of tests on the model.

**To test:**
- Try to submit a custom command, for example the default (the reconstruction script with --help), or any other simple command on SCARF. You shouldn't get any error.
- Check code changes (fix and unit tests to cover regressions like this).

Fixes #17217.

Added as small release note: https://github.com/mantidproject/mantid/commit/5dc9bcfe6a7aca2409d393e8cb9f2eea0a03cba1

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
